packages/dotlottie-js/src/node/lottie-animation.ts

```diff
@@ -29,18 +30,17 @@ export class LottieAnimation extends LottieAnimationCommon {
   }
 
   /**
-   * to do : Support more than image assets (fonts...)
    *
    * Extract image assets from the animation.
    *
    * @returns boolean - true on error otherwise false on success
    */
   protected override async _extractImageAssets(): Promise<boolean> {
-    if (!this._data) throw createError('Asset extraction failed.');
+    if (!this._data) throw new DotLottieError('Asset extraction failed.');
```

Comment:
Do you think we need to have more explicit error messages ? 

I feel that `Asset extraction failed.` doesn't describe the reason why it failed. 

Probably something like `Failed to extract image assets: Animation data does not exist.` might give clearer insights into the root cause.